deploy/template.yaml

```diff
@@ -1163,7 +1162,9 @@ Resources:
           CachedMethods:
```

Comment:
The documentation on this parameter is pretty appalling: https://docs.aws.amazon.com/cloudfront/latest/APIReference/API_CachedMethods.html

Do we need to specify this at all (it's not a required param on CloudFront CloudFormation: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cloudfront-distribution-cachebehavior.html#cfn-cloudfront-distribution-cachebehavior-cachedmethods)